openchemistry/io/orca.py

```diff
@@ -0,0 +1,121 @@
+import json
+import openchemistry as oc
+from avogadro.core import Molecule
+from avogadro.io import FileFormatManager
+from jsonpath_rw import parse
+from .base import BaseReader
+from .constants import EV_TO_J_MOL
+from .utils import _cclib_to_cjson_basis
+
+
+def OrcaReader(BaseReader):
+    """A class to parse orca output files and dump a chemical json file"""
+
+    def read(self, output_file):
```

Comment:
The signature of this method does not conform with what the abstract base class requires: `def read(self)`.
The reader holds a reference to a file like object, which is passed to it in the constructor.
Which also means that any of the code below that uses `output_file` will need to be changed.

Reply:
I solved this.